### PR TITLE
Add default templates for issue

### PR DESCRIPTION
**Pull request by h00130372**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+If the bug is related a CI job, put the job information below:
+* job name:
+* job build status link:
+* OpenStack release against with:
+
+### Target Project information
+* target project name:
+* version:
+
+### To Reproduce
```
Comment (mrhillsman) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
### Steps to reproduce the issue
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Steps to reproduce the behavior:
```
Comment (mrhillsman) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion

```

File: .github/ISSUE_TEMPLATE/integration_test_request.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+If you are interested in access to OpenLab resources, please fill out the details below. You can always find more information about OpenLab at https://openlabtesting.org
```
Comment (mrhillsman) [inline .github/ISSUE_TEMPLATE/integration_test_request.md]: Entire file needs to change to the following; simple copy and paste is sufficient:

If you are interested in access to OpenLab resources, please fill out the details below. You can always find more information about OpenLab at https://openlabtesting.org

If you are just filing an issue, ignore/delete these fields and file your issue.

### Full Name

### Email

### Company/Organization

### Job Title

### Phone

### What is your focus?
- [ ] Performance
- [ ] Scale
- [ ] Reliability
- [ ] Resiliency
- [ ] Integration
- [ ] Multi-Cloud
- [ ] Operational Tooling
- [ ] Edge or IoT
- [ ] Machine Learning
- [ ] Artificial Intelligence
- [ ] Application Ecosystem
- [ ] Other


### If this is for an open source project what is it?

### Brief project description

### Is project code 100% open source? If so, what is the URL or URLs where it is located?

### What kind of machines (VMs or Baremetal) and how many do you expect to use?

### What OS are you planning to use?

### Any special network configuration you expect or anticipate implementing?

### Any architecture or other specifications/requirements (CPU, RAM, GPU, etc)?

### Please state your contributions to the open source community and any other relevant initiatives.

### What testing are you planning to implement or need assistance implementing?

### How will this testing advance application and/or tooling built on-top of open infrastructure?

### Will you publish blog or paper from your testing?

- [ ] Yes
- [ ] No - if no, how will you present your results?

### Any other relevant details we should know about while preparing the infrastructure?

Comment (kiwik) [reply .github/ISSUE_TEMPLATE/integration_test_request.md]: 1. Yes, reporting bug and submitting test request maybe come from different actual role, so split to two templates to make new member focus on thing they care about.
2. We should add Github account id into joining template in order to we can use it to invite new member into OpenLab github organization.
3. Please mark `job title` and `phone` as optional, I think not everybody would like to show this personal information.

File: .github/ISSUE_TEMPLATE/joinin.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+Here are some ways to support OpenLab:
+
+* Additional infrastructure (commodity hardware, cloud resources, etc)
+* Engineering support (developers, project management, software support)
+* QA, QE, and test (unit, integration, e2e, scenario) builders.
+* Software/application and tooling integrations
+* Operations support
+
+If you would like to join us, please leave your contact information below, we will invite you into OpenLab Team later. As a member you will find out what's happening in OpenLab, what fellow open source community members are doing using OpenLab resources, also you can access OpenLab resources for your own testing.
```
Comment (kiwik) [inline .github/ISSUE_TEMPLATE/joinin.md]: s/Team/team/

File: .github/ISSUE_TEMPLATE/joinin.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+Welcome to OpenLab!
+
+Here are some ways to support OpenLab:
+
+* Additional infrastructure (commodity hardware, cloud resources, etc)
+* Engineering support (developers, project management, software support)
+* QA, QE, and test (unit, integration, e2e, scenario) builders.
+* Software/application and tooling integrations
+* Operations support
```
Comment (kiwik) [inline .github/ISSUE_TEMPLATE/joinin.md]: Please remove this line, and put `Additional infrastructure` at here

File: .github/ISSUE_TEMPLATE/joinin.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+If you would like to join us, please leave your contact information below, we will invite you into OpenLab Team later. As a member you will find out what's happening in OpenLab, what fellow open source community members are doing using OpenLab resources, also you can access OpenLab resources for your own testing.
+
+### Full Name
+
+### Email
+
+### Organization
```
Comment (kiwik) [inline .github/ISSUE_TEMPLATE/joinin.md]: 1. Github account id
2. IRC nickname (Optional)
3. Company/Organization
4. Job Title (Optional)
5. Phone (Optional)

File: .github/ISSUE_TEMPLATE/resource_donating.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+Thank you for your donation!
```
[the remark this answers is not in the document]
Comment (liu-sheng) [reply .github/ISSUE_TEMPLATE/resource_donating.md]: hmm.. this long sentence is obscure for me to understand, my poor english :)

File: .github/ISSUE_TEMPLATE/resource_donating.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+- [ ] Yes
+- [ ] No
+
+### If no, how to provide recognition?
+
+### Interested in sponsoring any of the following?
+- [ ] Additional Infrastructure
+- [ ] Engineering Support
+- [ ] Product/Project Manager(s)
+- [ ] QA, QE, and test builders
```
Comment (liu-sheng) [inline .github/ISSUE_TEMPLATE/resource_donating.md]: - [ ] Engineering Support
- [ ] Product/Project Manager(s)
- [ ] QA, QE, and test builders

these look like a bit duplicated with the `joinin` section.

File: .github/ISSUE_TEMPLATE/joinin.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
+Welcome to OpenLab!
+
+Here are some ways to support OpenLab:
```
Comment (liu-sheng) [inline .github/ISSUE_TEMPLATE/joinin.md]: maybe need a more detailed introduction about to becoming a member of OpenLab teams, "some ways to support OpenLab" is too broad description. the can also include donation, integration testing...

File: .github/ISSUE_TEMPLATE/joinin.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+
+* Additional infrastructure (commodity hardware, cloud resources, etc)
```
Comment (liu-sheng) [inline .github/ISSUE_TEMPLATE/joinin.md]: this should be in donation

File: .github/ISSUE_TEMPLATE/joinin.md (hunk 5 above)
Comment (liu-sheng) [inline .github/ISSUE_TEMPLATE/joinin.md]: need to clearly classify `joinin`, `integration`, `donation`

File: .github/ISSUE_TEMPLATE/integration_test_request.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+
+### What kind of machines (VMs or Baremetal) and how many do you expect to use?
+
+### What OS are you planning to use?
+
+### Any special network configuration you expect or anticipate implementing?
+
+### Any architecture or other specifications/requirements (CPU, RAM, GPU, etc)?
+
+### Please state your contributions to the open source community and any other relevant initiatives.
```
Comment (liu-sheng) [inline .github/ISSUE_TEMPLATE/integration_test_request.md]: not sure what this is

File: .github/ISSUE_TEMPLATE/joinin.md (hunk 6 above)
Comment (h00130372) [reply .github/ISSUE_TEMPLATE/joinin.md]: I think email is needed.

Comment (kiwik) [reply .github/ISSUE_TEMPLATE/joinin.md]: Yes, it is

File: .github/ISSUE_TEMPLATE/resource_donating.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
+Thank you for your donation!
+
+Leave your contact information below, our marketing will contact you later.
+
+### Full Name
```
Comment (kiwik) [inline .github/ISSUE_TEMPLATE/resource_donating.md]: remove

File: .github/ISSUE_TEMPLATE/resource_donating.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+
+### Email
```
Comment (kiwik) [inline .github/ISSUE_TEMPLATE/resource_donating.md]: remove

File: .github/ISSUE_TEMPLATE/resource_donating.md [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+
+### Organization
```
Comment (kiwik) [inline .github/ISSUE_TEMPLATE/resource_donating.md]:
```suggestion
### Donors company/organization
```

File: .github/ISSUE_TEMPLATE/resource_donating.md [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
+
+### Job Title
```
Comment (kiwik) [inline .github/ISSUE_TEMPLATE/resource_donating.md]: remove

File: .github/ISSUE_TEMPLATE/resource_donating.md [hunk 16]
```diff
@@ -0,0 +1,32 @@
...
+
+### Phone
```
Comment (kiwik) [inline .github/ISSUE_TEMPLATE/resource_donating.md]: remove

**Pull request merged**